guides/spin/pipeline/index.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: single
+title:  "Pipeline"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Overview
+
+Once you have `spin` install and configured, you can use it to start
+managing your Spinnaker applications.
+
+`spin` can manage the whole lifecycle of your pipeline:
+
+```bash
+# NOTE: You can inspect any command with the `--help` flag.
+$ spin pipeline --help
+
+This command is accessed by using one of the subcommands below.
+
+Subcommands:
+    delete     Delete the provided pipeline.
+    execute    Execute the provided pipeline.
+    get        Get the pipeline with the provided name from the provided application.
+    list       List the pipelines for the provided application.
+    save       Save the provided pipeline.
+
+```
+
+The following assumes Spinnaker is running and Gate is
+listening on `http://localhost:8084`. If gate is running elsewhere,
+you can set the Gate endpoint with the global `--gate-endpoint` flag.
+
+## Managing Your Pipeline's Lifecycle
+
+### Save
```

Comment:
As with the Applications doc, delete the first sentences of these and work them into the heading.